build.gradle

```diff
@@ -11,16 +11,15 @@ plugins {
 	id 'eclipse'
 	id 'maven-publish'
 	id 'checkstyle'
-	id 'com.diffplug.spotless' version "6.12.0"
-	id 'fabric-loom' version '1.2-SNAPSHOT' apply false
+	id 'com.diffplug.spotless' version "6.22.0"
```

Comment:
This spotless version (Required for Gradle 8.3) depends on Java 11. The only way forward is to only build on Java 11+ (Likely just go to 17).

Loader can still target Java 8 without issue.